### ISSUE TITLE
Add rig control to TS890

### ISSUE THREAD

**Issue by ny4i** [n4af/TR4W#436]
This radio resembles the TS990 more than any other so reference the issues with #435.

**Comment by ny4i** [comment]
https://www.kenwood.com/i/products/info/amateur/pdf/ts890_pc_command_en_rev1.pdf

**Comment by ny4i** [comment]
This radio also supports a network connection. See this doc for more info and implementation details: https://github.com/ny4i/TR4QT/blob/master/docs/kenwood-direct-connection-flow.md

**Comment by ny4i** [comment]
Testing report from @n2skh

Tom, I finally had some time today to test TR4W with the 890.  Since I'd been using the USB connection with the Kenwood remote software (ARCP-890), I first reconfigured it to use TCP/IP and confirmed the radio CAT interface was working, then I installed and configured TR4W.  When TR4W started the radio appeared to connect, but my 890 immediately gave me a message on the screen that the bandscope was out of range. It then blanked the screen after a few seconds and put up a message to turn off the DC power, which I did.  When I turned it back on, the bandscope error was still there (and I noticed the bandscope range was from 0 to 0).  I was able to clear the message and restore the bandscope by hitting the scope range button. (edited) 
[11:15 PM]I tried starting TR4W again, but this time I turned off the bandscope on the radio first and I didn't get that same message, but the screen would blank and reset every 5 to 10 seconds (I didn't time it), and I noticed the radio 1 indicator on the main TR4W screen would go from black to red and then back to black when this happened, so I guessed the connection was being lost and reconnected.  I confirmed this by looking at the log. (edited) 
[11:16 PM]I never did get the frequency, etc from the radio to display on TR4W, so even though it's connecting, it's not quite right yet.
[11:18 PM]Here's the log file for one connection cycle (without the bandscope turned on).

[tr4w.log](https://github.com/user-attachments/files/27971926/tr4w.log)

**Comment by ny4i** [comment]
## Field test report — N2SKH, 2026-05-28 (v4.147.15-era build)

Ron N2SKH tested LAN control against his TS-890S at 192.168.50.195:60000 with `DEBUG LOG LEVEL = TRACE`. Reconnect loop is stable but the session never reaches CAT.

### Observed pattern (repeats every cycle)

1. TCP connect → OK
2. TX `##CN;` → RX `##CN1;` (radio requests credentials)
3. TX `##ID00507N2SKH<pw>;` → RX `##ID1;` (credentials accepted)
4. **Total silence from the radio for ~21 s**
5. Radio sends FIN (`[Disconnected.]` — graceful close)
6. Reconnect, repeat forever

State machine sits in `ksWaitingForTI` indefinitely. `PollRadioState` (which sends the `PS;` 5 s keepalive) early-returns until `ksAuthenticated`, so we send nothing while waiting for `##TI1;` and the radio idles us out.

### Ruled out

- **External CAT not routed to LAN.** Ron runs Kenwood ARCP-890 over the same LAN port against the same radio and it works fine. Auth path + LAN routing are correctly configured at the radio.
- **Missing keepalive/##TI-wait fix.** Commit `21df4a1` is already in Ron's build (the "awaiting ##TI handshake" log line is the tell).
- **Menu 7-15.** Ron's current TS-890 firmware no longer exposes menu 7-15; LAN-related items are now under Advanced (IP config only). Not the cause — ARCP works.

### Cosmetic side-finding

`tr4w/src/uRadioPolling.pas:744` and `:746` contain literal `EF BF BD` bytes (UTF-8 U+FFFD replacement char) where an arrow glyph was lost during an editor save. Shows up as `�` in the log:
```
[pNetworkRadio] TS-890S � alert color ON
```
Pure cosmetic, unrelated to this bug. One-line cleanup.

### Working hypotheses (in priority order)

1. **Radio doesn't push `##UE1;`/`##TI1;` unsolicited on this firmware** — needs a client probe (e.g. `##UE;`) or its first CAT command to elicit them. ARCP starts CAT polling immediately after `##ID1;`, which masks this.
2. **Framing mismatch** — radio sends `##UE1;`/`##TI1;` but in a length-prefixed binary envelope that our `;`-terminated reader drops on the floor.
3. **Stale TCP half-state** — `##TI1;` only on the first login of a power-cycle; subsequent reconnects don't re-send.

### Next step — waiting on data, no code changes

Ron is capturing a pcap. Asked for two captures back-to-back if possible:
- TR4W session (reproduces the hang)
- ARCP-890 session against same radio (gold-standard reference)

Wireshark filter: `tcp port 60000`. Both captures will show definitively whether the radio sends anything between `##ID1;` and FIN, and how ARCP differs in what it sends in that same window.

### Likely fix shape (deferred until pcap)

In `uRadioKenwoodTS890.pas`:
- Transition `ksWaitingForTI → ksAuthenticated` either on `##TI1;` or after a short timeout (~1.5 s), and start `PS;` keepalive + initial state queries immediately.
- Optionally probe with `##UE;` on entering `ksWaitingForTI`.

But the pcap may change this — if framing turns out to be the issue (hypothesis 2), the reader needs the fix, not the state machine.
